resources/js/components/items/FeedbackForm.vue

```diff
@@ -140,7 +151,24 @@ export default {
             this.locations = data;
         },
 
-        async createFeedback() {
+        handleError(err){
+            let data = err.response.data;
+            this.$refs.modalWrapper.errors = data.errors;
+        },
+
+        handleSuccess(response){
+            this.$refs.modalWrapper.closeModal();
+            
+            this.$emit('blank');
+            
+            this.resetData();
+            
+            setTimeout(function(){
+                this.$emit('created', response.data);
+            }.bind(this), 1000);
```

Comment:
Peço desculpas por essa gambiarra, mas não tive muitas alternativas. Atualmente com o tratamento de erros não há _delay_ para o retorno do item quando esse é criado com sucesso.
Por isso tive que simular esse _delay_ com essa função (:vomiting_face:), é totalmente forçado mas infelizmente tive que apelar :disappointed:

Reply:
Nem se estressa com isso que é bem comum ter esse tipo de delay em UI. A diferença de usar Vue e React é que eles animam esse delay, mas ele está lá 😄